### ISSUE TITLE
Glium doesn't crash or give error when loading an invalid opengl version

### ISSUE THREAD

**Issue by TyOverby** [tomaka/glium#328]
When I have `.with_gl_version((3,2))` in my window builder, and then I run `display.assert_no_error()` I get an error `GL_INVALID_ENUM`.  

`glxinfo | grep OpenGL` :

```
OpenGL vendor string: NVIDIA Corporation
OpenGL renderer string: GeForce GTX 285/PCIe/SSE2
OpenGL core profile version string: 3.3.0 NVIDIA 331.113
OpenGL core profile shading language version string: 3.30 NVIDIA via Cg compiler
OpenGL core profile context flags: (none)
OpenGL core profile profile mask: core profile
OpenGL core profile extensions:
OpenGL version string: 3.3.0 NVIDIA 331.113
OpenGL shading language version string: 3.30 NVIDIA via Cg compiler
OpenGL context flags: (none)
OpenGL profile mask: (none)
OpenGL extensions:
```

**Comment by tomaka** [comment]
Which OpenGL version did you "load"?

**Comment by TyOverby** [comment]
> When I have .with_gl_version((3,2))

**Comment by tomaka** [comment]
I added debug output, could you run it again to see if there's a difference?

**Comment by tomaka** [comment]
Whatever this was, it was very likely fixed by #782
